LanguageFeatures/Wildcards/super_parameters_A04_t01.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Multiple `super._` and `this._` parameters can occur in the same
+/// constructor without any name collision.
+///
+/// @description Checks that multiple `super._` and `this._` parameters can
+/// occur in the same constructor without any name collision.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=wildcard-variables
+
+import '../../Utils/expect.dart';
+
+class A {
+  final int x, y;
+  A(this.x, this.y);
```

Comment:
Shouldn't we have enough parameters to avoid extra errors?:
```suggestion
  final int x, y;
  A(this.x, this.y, int _);
```

Reply:
Oops, you're right, sorry!